components/EventItem.tsx

```diff
@@ -1,326 +1,176 @@
+  const shadowColor = "#bdbdbd"
+  const lightEventColor = tinycolor(event.colorHex).lighten(10).toString()
 
-  const setTime = () => {
-    if (event.startTime) {
-      const date = new Date()
-      const diffTime = Math.abs(event.startTime.getTime() - date.getTime())
-      const diffMin = Math.ceil(diffTime / (1000 * 60))
-      const diffHours = Math.floor(diffMin / 60)
-      const diffDays = Math.floor(diffHours / 24)
-
-      if (diffMin < 60) {
-        setIsMin(true)
-        setIsHours(false)
-        setIsDays(false)
-        setTimeUntil(diffMin)
-      } else if (diffHours < 24) {
-        setIsMin(false)
-        setIsHours(true)
-        setIsDays(false)
-
-        if (diffMin % 60 >= 20 && diffMin % 60 <= 40) {
-          setTimeUntil(diffHours + 0.5)
-        } else {
-          setTimeUntil(diffHours)
-        }
-      } else {
-        setIsMin(false)
-        setIsDays(true)
-        setIsHours(false)
-        setTimeUntil(diffDays)
-      }
-    }
+  const onPressMore = () => {
+    return null
   }
 
-  useEffect(() => {
-    setTime()
-  }, [event.startTime])
-
-  const handleRequestToJoin = () => {
-    if (requested) {
-      setRequested(false)
-      setNumInvitations(numInvitations - 1)
-    } else {
-      setRequested(true)
-      setNumInvitations(numInvitations + 1)
-    }
-  }
+  return (
+    <Shadow
+      distance={5}
+      startColor={shadowColor}
+      offset={[0, 3]}
+      style={{ alignSelf: "stretch" }}
+    >
+      <View style={[styles.container]}>
+        {/* Profile Image, Name, More button */}
+        <View style={[styles.topRow, styles.flexRow]}>
+          <Image
+            style={styles.image}
+            source={require("../assets/icon.png")}
+            accessibilityLabel="profile picture"
+          />
+          <Text style={styles.name}>{event.username}</Text>
+          <IconButton
+            iconName={"more-horiz"}
+            style={styles.moreButtonStyle}
+            size={26}
+            onPress={onPressMore}
+          />
+        </View>
 
-  const displayTime = () => {
-    if (isDays) {
-      return `${timeUntil}d`
-    } else if (isHours) {
-      if (timeUntil === 1) {
-        return `${timeUntil}hr`
-      } else {
-        return `${timeUntil}hrs`
-      }
-    } else {
-      return `${timeUntil}min`
-    }
-  }
+        {/* Event Title, Location, Time */}
+        <View style={styles.middleRow}>
+          <Text style={styles.titleText}>{event.title}</Text>
 
-  return (
-    <View style={styles.secondaryContainerStyle}>
-      <View style={[styles.nestedReply]}>
-        {/* Header (Event Icon, Event Title, Distance) */}
-        <View style={[styles.flexRow, styles.eventContainerStyle]}>
-          <View style={[styles.spacingTop, { paddingLeft: "3%" }]}>
-            <Icon name="location-pin" color={event.colorHex} />
+          <View style={[styles.location, styles.flexRow]}>
+            <Icon name="location-on" color={event.colorHex} />
```

Comment:
These rows could also be extrapolated into their own component similar to the IconButton. Hold off on that for now though until we finalize the designs and know for sure whether we'll reuse this style in the future.